force-app/main/default/classes/JobApplicationHandlerTest.cls

```diff
@@ -112,4 +112,225 @@ public with sharing class JobApplicationHandlerTest {
             }
         }
     }
+
+    @IsTest
+    static void testTaxWithholdingCalculator(){
+    
+        List<Job_Application__c> jobAppList = new List<Job_Application__c>();
+        Job_Application__c newJobApp1 = new Job_Application__c (Salary__c = 10000, Salary_Type__c = 'Annual', Tax_Filing_Status__c = 'Single', Deduction_Type__c = 'None');
```

Comment:
What if we add this to the TestDataFactory class? I'm already creating hundreds of records in mass with that.

Reply:
@rileymeye1 I'm fine doing that. I'll take a look at the TestDataFactory and see how much work I need to do for that.